atd-vze/package.json

```diff
@@ -50,7 +49,6 @@
     "react-datepicker": "^2.9.6",
     "react-dom": "^16.8.6",
     "react-map-gl": "^7.1.7",
-    "react-map-gl-geocoder": "^2.0.11",
```

Comment:
Forgot to remove this in https://github.com/cityofaustin/atd-vz-data/pull/1399 🗑️